### PR TITLE
Problem: Missing ios build setting for simple demo (fix cronos-labs/play-unreal-plugin#206)

### PR DESCRIPTION
**Pull request by damoncro**

### CODE REVIEW

**Comment by damoncro** [comment]
Presubmission, only valid after https://github.com/cronos-labs/play-unreal-plugin/pull/205 is reviewed and merged.

File: README.md [hunk 1]
```diff
@@ -2,45 +2,36 @@
...
+- Plugin: Cronos Play For Unreal Engine in Epic Marketplace
+
+#### Features
+- Restore Wallet by Mnemonics, and generate address with Wallet APIs
+- Query details of ERC20/ERC721/ERC1155 tokens
+- Connect wallet with WalletConnect, by QR code or deep link
+- Sign personal message and transaction with WalletConnect
+
+### [play2earn](https://github.com/cronos-labs/play-unreal-demo/tree/main/play2earn)
+This demo shows the basic ERC721/ERC1155 usages on a first-character shooting game.
```
Comment (damoncro) [inline README.md:24]: @leejw51crypto Is that good enough?

**Review by leejw51crypto — APPROVED** [review]
lgtm

**Comment by damoncro** [comment]
It is ready to be merged, but it only works on local plugin, because the marketplace plugin is still unreleased.

**Review by tomtau — APPROVED** [review]
when will the new plugin version be published?

**Comment by damoncro** [comment]
Update the simple demo with local plugin instead, because only local plugin can be build on CI.

**Comment by damoncro** [comment]
> when will the new plugin version be published?

**Pull request merged**